server/internal/toolsets/impl.go

```diff
@@ -514,3 +541,131 @@ func (s *Service) RemoveOAuthServer(ctx context.Context, payload *gen.RemoveOAut
 
 	return toolsetDetails, nil
 }
+
+// Temporary method to create toolset versions alongside the existing toolset tools tracking
```

Comment:
Maybe just a little more in the comment about how the additional DB queries will need to be removed when backfill is completed